lib/util.js

```diff
@@ -37,3 +37,13 @@ exports.sorter = (a, b) => {
 
 	return processCompare(a, b);
 };
+
+exports.getPortFromPid = (val, list) => {
```

Comment:
I extracted this method from `fkill-cli` but it seems quite slow. For every process, we loop through the entire PIDs collection. This means that, in algorithm notation, it runs at time complexity `O(n^2)` (as far as my algorithm skills go). Wouldn't it be better if we just swapped the key-value pair from the PIDs map?

Something like this
```js
const swapped = new Map(Array.from(pids.keys()).map(port => [pids.get(port), port])));
```

Then we could just iterate over all the processes and write `swapped.get(process.pid)` which results more in a time complexity of `O(n)` as we don't have a nested loop.